app/services/transaction_search/searcher.rb

```diff
@@ -32,7 +32,7 @@ def self.search(order_details, params)
     end
 
     # Shorthand method for searchers used on the billing page
-    def self.billing_search(order_details, params, include_facilities = false)
+    def self.billing_search(order_details, params, include_facilities:)
```

Comment:
I don't have a strong feeling either way, but this could be optional with a default of `false`.